packages/next/src/build/webpack/loaders/metadata/discover.ts

```diff
@@ -11,7 +10,7 @@ const METADATA_TYPE = 'metadata'
 
 export const METADATA_RESOURCE_QUERY = '?__next_metadata'
 
-const staticAssetIconsImage = {
+export const STATIC_METADATA_IMAGES = {
```

Comment:
Maybe add `as const` to this so that it becomes readonly and the type is narrowed.

Reply:
will add in the later PR 👍